plugins/artifacts/artifacts.go

```diff
@@ -116,9 +116,9 @@ func (tp taskPlugin) errorHandled(name string, expires tcclient.Time, err error)
 	return false
 }
 
-func (tp taskPlugin) createUploadHandler(name, prefix string, expires tcclient.Time) func(string, ioext.ReadSeekCloser) error {
+func (tp taskPlugin) createUploadHandler(name string, expires tcclient.Time) func(string, ioext.ReadSeekCloser) error {
 	return func(path string, stream ioext.ReadSeekCloser) error {
-		return tp.attemptUpload(stream, path, filepath.Join(name, filepath.Base(path)), expires)
+		return tp.attemptUpload(stream, path, filepath.Join(name, path), expires)
```

Comment:
use `path` not `filepath` which is platform specific.. .This should always use slashes...